test/lib/tpunit++.cpp

```diff
@@ -316,6 +322,13 @@ int tpunit::_TestFixture::tpunit_detail_do_run(const set<string>& include, const
         }
     }
 
+    // Print message for each include pattern that did not match any test
+    for (const auto& pattern : include) {
+        if (includeMatched.find(pattern) == includeMatched.end()) {
```

Comment:
You can use `if(!includeMatched.contains(pattern)) {` in C++20

Reply:
Gonna take as NAB and someone can probably cleanup all `find` usage. Thanks.